features/nationaliteithistorie.feature

```diff
@@ -3,128 +3,441 @@
-    | 1      | 04        | 0001  | 057   |       | 20160803 |
-    | 2      | 54        |       |       | 403   | 20170526 |
-    | 2      | 54        |       |       | 402   | 20170210 |
-    | 2      | 54        | 0449  | 301   |       | 19930419 |
-
-  En de ingeschreven persoon met burgerservicenummer 999992466 kent de volgende nationaliteiten:
-    | Stapel | Categorie | 05.10 | 63.10 | 64.10 | 85.10    | 84.10 |
-    | 1      | 04        | 0057  | 301   |       | 20160601 |       |
-    | 1      | 54        | 0057  | 301   |       | 20160501 |       |
-    | 1      | 54        | 0057  | 301   |       | 20160401 |       |
-    | 1      | 54        | 0057  | 301   |       | 20160301 | O     |
-
-  En de ingeschreven persoon met burgerservicenummer 999992855 kent de volgende nationaliteiten:
-    | Stapel | Categorie | 05.10 | 63.10 | 64.10 | 85.10    | 84.10 |
-    | 1      | 04        | 0001  | 018   |       | 20041015 |       |
-    | 2      | 04        |       |       |       | 20121022 |       |
-    | 2      | 54        | 0251  |       |       | 20121022 | O     |
-    | 3      | 04        | 0445  |       |       | 19611230 |       |
-    | 4      | 04        | 0446  |       |       | 19611230 |       |
-
-  Scenario: actuele en beëindigde nationaliteiten in de juiste volgorde
-    Als de nationaliteithistorie met burgerservicenummer 999991292 wordt geraadpleegd
-    Dan bevat het antwoord 3 voorkomens
-    En wordt de nationaliteithistorie teruggegeven in de volgorde en met waarden:
-    | # | nationaliteit.code | aanduidingBijzonderNederlanderschap | datumIngangGeldigheid | datumTot             | indicatieNationaliteitBeeindigd | redenBeeindigen.code |
-    | 0 | 0001               | -                                   | 1991-02-01            | -                    | -                               | -                    |
-    | 0 | 0057               | -                                   | 1983-12-13            | 2014-06-01           | true                            | 404                  |
-    | 0 | 0100               | -                                   | 1989-03-01            | 1994-06-01           | true                            | 404                  |
-
-  Scenario: meerdere actuele en beëindigde nationaliteiten
-    Als de nationalteithistorie met burgerservicenummer 999990457 wordt geraadpleegd
-    Dan bevat het antwoord 5 voorkomens
-    En wordt de nationaliteithistorie teruggegeven in de volgorde en met waarden:
+  # in onderstaande scenario's staat ten behoeve van de provider implementatie in Gegeven achter de veldnaam tussen haakjes de veldcode zoals deze in de bron volgens LO GBA is geregistreerd
+
+  Rule: In het antwoord wordt indicatieNationaliteitBeeindigd opgenomen met de waarde true, wanneer in de actuele nationaliteit (categorie 04) GEEN nationaliteit (05.10) noch aanduiding bijzonder Nederlanderschap (65.10) is opgenomen, of wanneer in categorie 04 reden beëindigen nationaliteit (64.10) is opgenomen.
```

Comment:
Als ik deze rule goed interpreteer, dan is indicatieNationaliteitBeëindigd true, of 05.10 is NIET ingevuld, of 65.10 is NIET ingevuld of 64.10 is gevuld. Klopt dit? Als ik door de scenario's scan, dan is 05.10 en 65.10 niet ingevuld als 64.10 is gevuld en omgekeerd. Volgens mij komt alle 3 niet ingevuld niet voor

Reply:
> dan is indicatieNationaliteitBeëindigd true, of 05.10 is NIET ingevuld, of 65.10 is NIET ingevuld of 64.10 is gevuld. Klopt dit?

Er staat indicatieNationaliteitBeeindigd=true als er GEEN 05.10 of 65.10 is of wanneer 64.10 wel

> Volgens mij komt alle 3 niet ingevuld niet voor

klopt. als het volgens Lo GBA is ingevuld komt óf 05.10 óf 65.10 óf 64.10 voor. ik vereenvoudig de regel dat indicatieNationaliteitBeeindigd=true als 64.10 voorkomt en voeg een scenario toe dat 64.10 waarde onbekend (000) heeft